apps/docs/src/app/platform/component-docs/platform-file-uploader/platform-file-uploader-examples/platform-file-uploader-reactive-example.component.html

```diff
@@ -0,0 +1,41 @@
+<form #upload="ngForm" (ngSubmit)="onSubmit(upload)">
+    <fdp-form-group [formGroup]="customForm">
+        <fdp-form-field
+            #ffl1
+            label="File Uploader Reactive"
+            id="input1"
+            name="reactiveFormInput"
+            zone="zLeft"
+            rank="1"
+            [validators]="requiredDateValidator"
```

Comment:
if we are validating for `required`, there should be a `required` attribute too. I don't see the validation work.